tests/integration/test_snap.py

```diff
@@ -65,7 +66,11 @@ def test_snap_refresh():
 def test_snap_set_and_get_with_typed():
     cache = snap.SnapCache()
     lxd = cache["lxd"]
-    lxd.ensure(snap.SnapState.Latest, channel="latest")
+    try:
```

Comment:
From Weii: this is to address instability in the GitHub Runners when running these tests.